stream_alert_cli_module/test.py

```diff
@@ -168,7 +161,7 @@ def check_keys(self, rule_name, test_record):
         Args:
             rule_name: The name of the rule being tested. This is passed in
                 here strictly for reporting any errors with key checks.
-            test_record [dict]: Test record metadata dict
+            test_record: [dict] Test record metadata dict
```

Comment:
Can we stick to [this style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/#type-annotations) going forward?